book/course/conda.md

```diff
@@ -350,6 +440,176 @@ data-sci-env               /home/home01/arcusers/.conda/envs/data-sci-env
+    scipy-1.9.1                |  py310hdfbd76f_0        26.2 MB  conda-forge
+    statsmodels-0.13.2         |  py310hde88566_0        11.2 MB  conda-forge
+    ------------------------------------------------------------
+                                           Total:        67.4 MB
+
+The following NEW packages will be INSTALLED:
+
+  libblas            conda-forge/linux-64::libblas-3.9.0-16_linux64_openblas
+  libcblas           conda-forge/linux-64::libcblas-3.9.0-16_linux64_openblas
+  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-12.1.0-h69a702a_16
+  libgfortran5       conda-forge/linux-64::libgfortran5-12.1.0-hdcd56e2_16
+  liblapack          conda-forge/linux-64::liblapack-3.9.0-16_linux64_openblas
+  libopenblas        conda-forge/linux-64::libopenblas-0.3.21-pthreads_h78a6416_3
+  libstdcxx-ng       conda-forge/linux-64::libstdcxx-ng-12.1.0-ha89aaad_16
+  numpy              conda-forge/linux-64::numpy-1.23.2-py310h53a5b5f_0
+  packaging          conda-forge/noarch::packaging-21.3-pyhd8ed1ab_0
+  pandas             conda-forge/linux-64::pandas-1.4.4-py310h769672d_0
+  patsy              conda-forge/noarch::patsy-0.5.2-pyhd8ed1ab_0
+  pyparsing          conda-forge/noarch::pyparsing-3.0.9-pyhd8ed1ab_0
+  python-dateutil    conda-forge/noarch::python-dateutil-2.8.2-pyhd8ed1ab_0
+  python_abi         conda-forge/linux-64::python_abi-3.10-2_cp310
+  pytz               conda-forge/noarch::pytz-2022.2.1-pyhd8ed1ab_0
+  scipy              conda-forge/linux-64::scipy-1.9.1-py310hdfbd76f_0
+  six                conda-forge/noarch::six-1.16.0-pyh6c4a22f_0
+  statsmodels        conda-forge/linux-64::statsmodels-0.13.2-py310hde88566_0
+
+
+Proceed ([y]/n)?
+```
+````
+
+Conda will always prompt the user if we're happy to proceed with the installation and specifies all the other packages that will be installed or updated that are required for our specified package.
+We confirm we wish to proceed by entering `y` and pressing Return.
+
+````{admonition} View full output
+:class: dropdown
+```
+Proceed ([y]/n)? y
+
+Downloading and Extracting Packages
+pytz-2022.2.1        | 224 KB    | ##################################### | 100%
+libopenblas-0.3.21   | 10.1 MB   | ##################################### | 100%
+scipy-1.9.1          | 26.2 MB   | ##################################### | 100%
+patsy-0.5.2          | 188 KB    | ##################################### | 100%
+statsmodels-0.13.2   | 11.2 MB   | ##################################### | 100%
+pandas-1.4.4         | 12.5 MB   | ##################################### | 100%
+numpy-1.23.2         | 7.1 MB    | ##################################### | 100%
+Preparing transaction: done
+Verifying transaction: done
+Executing transaction: done
+```
+````
+
+This installs any packages that are currently not installed (Conda caches packages locally incase they are required by other packages, this speeds up installs but uses more disk space to maintain this cache).
```

Comment:
```suggestion
This installs any packages that are currently not installed (Conda caches packages locally in case they are required by other packages, this speeds up installs but uses more disk space to maintain this cache).
```